metadata/overlays/istio/virtual-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ui
```

Comment:
As I was chatting with @rileyjbauer the other day, I think we need to rename this to `metadata-ui` because it does not appear that the `namePrefix: metadata-` from the base  kustomization.yaml file gets applied here and the prefix definitely helps to disambiguate this from other UI services.